docs/how-to-guides/install-seven-add-on.md

```diff
@@ -0,0 +1,189 @@
+---
+myst:
+  html_meta:
+    "description": "How to install a Seven frontend add-on with Cookieplone"
+    "property=og:description": "How to install a Seven frontend add-on with Cookieplone"
+    "property=og:title": "Install Seven with Cookieplone"
+    "keywords": "Plone, Plone 6, Volto, create, project, install, Cookieplone"
+---
+
+
+(install-cookieplone-label)=
+
+# Install Seven with Cookieplone
+
+This chapter describes how you can create a web application using the {term}`Cookieplone` template.
+
+This template is the recommended way to start a new add-on using the Seven frontend.
+It also includes tools for development and deployment.
+
+```{seealso}
+For other installation options, see {ref}`get-started-install-label`.
+```
+
+(create-project-cookieplone-system-requirements)=
+
+## System requirements
+
+Plone and Seven have both hardware requirements and software prerequisites.
```

Comment:
Don't switch context between "Plone" and "Plone and Seven". Also the latter makes it sound as if they are two separate products, when we don't yet know.